notes/stories/film/film-scifi-2010s.st.txt

```diff
@@ -4148,6 +4148,48 @@ choosing whether to kill someone to spare them suffering [Nathan was on the verg
 meteor [A dazzling colored, but foul smelling meteor fell on the Gardner's property.]
 
 
+movie: Gemini Man (2019)
+========================
+
+:: Title
+Gemini Man
+
+:: Description
+Gemini Man is a 2019 American action thriller film directed by Ang Lee.
+
+Synopsis: A retiring hitman is targeted by a much younger clone of himself
+while on the run from the government.
+
+:: Date
+2019-10-01
+
+:: References
+https://en.wikipedia.org/wiki/Gemini_Man_(film)
+
+:: Ratings
+3 <paul>
+
+:: Choice Themes
+what if I met my double [The main novelty of the story surrounds the 51-year-old, over-the-hill assassin Henry Brogan being pursued by his younger and seemingly more ruthless clone, Junior. And while not explicitly mentioned, the file title "Gemini Man" evidently draws a parallel between the clones Henry/Junior and the Gemini twins Castor/Pollux from Greek mythology.]
```

Comment:
Judgment call I guess, but since you describe him as "a younger version" it doesn't sound like a "perfect double" to me. Slippery slope? Also aspects of this is mixed in:

https://www.themeontology.org/theme.php?name=what%20if%20I%20met%20myself%20from%20a%20different%20time

Perhaps a better choice?

Reply:
Changed it to *what if I met myself from a different time* without hesitation.

Notes:
- The current *what if I met my double* usages are a mixed bag.
- It occurred to me what we're missing is a general *what if I met a version of myself* theme to use in these and other somewhat borderline cases.